src/System.Management.Automation/utils/Telemetry.cs

```diff
@@ -83,6 +83,11 @@ internal enum TelemetryType
         /// Remote session creation.
         /// </summary>
         RemoteSessionOpen,
+
+        /// <summary>
+        /// Send telemetry for a feature when used.
+        /// </summary>
+        FeatureUse,
```

Comment:
What is the different between this one and `ExperimentalFeatureUse`? It would be helpful to update the comment to call out the difference.

Reply:
by using a different api, the telemetry has a different name in the telemetry back end for easier querying. I'll add clarifying language in this area